### PR TITLE
Update README setup instructions

### PR DESCRIPTION
**Pull request by yangshun**
Hi there, Docusaurus maintainer here. Was looking into how Babel was using Docusaurus and ran into some issues with setting up (due to outdated instructions). Have updated the instructions in this PR.

### Test Plan

1. Re-clone the repository 
1. Following the updated instructions for setting up
1. Successfully ran the site locally

### CODE REVIEW

**Comment by hzoo** [comment]
Deploy preview for *babel-new* ready!

Built with commit 2f3748e4ada8fadbc071bce0f491ab37c7fcf783

https://deploy-preview-1635--babel-new.netlify.com

**Comment by hzoo** [comment]
Deploy preview for *babel-new* ready!

Built with commit 71819ad232c61ab3c9f46ad8f9bb865c2faac8ea

https://deploy-preview-1635--babel-new.netlify.com

File: README.md [hunk 1]
```diff
@@ -11,11 +11,13 @@ This is the source for the [babeljs.io](https://babeljs.io) website: feel free t
 
 ```bash
 $ git clone git@github.com:babel/website.git
-$ cd website
```
[the remark this answers is not in the document]
Comment (yangshun) [reply README.md]: Oh yes I see. I was confused. Do we need two `cd website` commands then?

[the remark this answers is not in the document]
Comment (yangshun) [reply README.md]: I'll close this then.

[the remark this answers is not in the document]
Comment (microbouji) [reply README.md]: Only one `cd <repo-name>` works. `npm start` in the root does the `cd website` for you.

**Pull request closed without merging**